src/test/java/com/itacademy/cms/service/UserServiceIntegrationTest.java

```diff
@@ -0,0 +1,91 @@
+package com.itacademy.cms.service;
+
+import com.itacademy.cms.exeption.UserNotFoundException;
+import com.itacademy.cms.model.User;
+import com.itacademy.cms.model.dto.UserDto;
+import com.itacademy.cms.model.enums.Role;
+import javax.transaction.Transactional;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+
+@SpringBootTest
+@Transactional(Transactional.TxType.REQUIRED)
+@ActiveProfiles(profiles = "test")
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+public class UserServiceIntegrationTest {
+
+  @Autowired
+  UserService userService;
+
```

Comment:
it's better to delete whole table before eact test, using Sprign Data delete() and `@BeforeEach`